### DISCUSSION TITLE
Indicator for start of new text when scrolling

### DISCUSSION THREAD

**Discussion by antrmn** [koreader/koreader#11996] [Q&A]
I'm reading a PDF with reflowed text on my Onyx Boox Page and when getting in the next page part of the previous page will still be visible:

https://github.com/koreader/koreader/assets/37816362/6eb5af85-dbd4-4b21-a4c2-7a1e4cf6a9a0

In this short video, I press the screen to get forward in the page but the new text appears at the 11th line in the screen. It is really confusing for me and every time I go forward in the document I need to find what's the beginning of the new text in order to resume my reading. Is there some way to aid this issue of mine? Maybe an indicator that appears near the start of the new "page"? TIA.

**Comment by jonnyl2** [comment]
There are 2 possible reasons why the overlap is that big:

1. You have View Mode set to `page` rather than `continuous`. `Page` mode only goes to the end of each page, even when Reflow is enabled. Therefore, with Reflow Mode enabled it is recommended that you change it to continuous. (Bottom menu; 'page' tab.)
2. Your `DOVERLAPPIXELS` setting is set too high. That is the number of pixels that are repeated on the top of each screen in the reader. You can decrese it in the Advanced settings: Open File browser, open top menu, Tools (3rd menu) -> More tools -> Advanced settings. Then set DOVERLAPPIXELS to something lower (I have mine set to 10).
